src/v/redpanda/application.cc

```diff
@@ -648,6 +654,14 @@ void application::wire_up_redpanda_services() {
           .invoke_on_all(&cluster::partition_manager::stop_partitions)
           .get();
     });
+
+    _deferred.emplace_back([this] {
+        nr_partition_manager
+          .invoke_on_all(
+            &cluster::non_replicable_partition_manager::stop_partitions)
```

Comment:
Is this not called `stop()` because you need to tweak the shutdown order?

Reply:
Yes, I modeled all of this after the existing `partition_manager`

Reply:
removed